### PR TITLE
NSDictionary+Parsing with unit testing added to project. BPScanResult parsing improved.

### PR DESCRIPTION
**Pull request by ArkadiuszBanas**
I've added new category to NSDictonary. Two new methods are used in BPScanResult+Utilities and replaced with handleNull:. Methods contain class verification. Both methods are 100% unit tests covered.

### CODE REVIEW

File: BuyPolish/Pola/Utilities/Categories/NSDictionary+Parsing.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//
+//  NSDictionary+Parsing.h
+//  Pola
+//
+//  Created by Arkadiusz Banaś on 14/02/16.
+//  Copyright © 2016 PJMS. All rights reserved.
+//
+
```
Comment (paweljaneczek) [inline BuyPolish/Pola/Utilities/Categories/NSDictionary+Parsing.h]: just please remove file comments. We don't use it in anywhere

**Comment by paweljaneczek** [comment]
Remove `[BPUtilities handleNull:]`. It is not used anymore

**Comment by paweljaneczek** [comment]
Great! thanks for PR :) :+1:

**Comment by ArkadiuszBanas** [comment]
Hi Paweł :) 
In that case - to be consistent  - I've removed header comments from all Pola files.

**Comment by paweljaneczek** [comment]
@ArkadiuszBanas great! I thought that there are no header comments ;) :+1:

**Pull request merged**